index.php

```diff
@@ -1,40 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-   <meta content="text/html"; charset=UTF-8" http-equiv="Content-Type" />
+   <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
 	<title>Multiplication table</title>
 </head>
 <body>
 <h1>Multiplication table</h1>
 
-<?php
-
-echo '<p>Hello fmi. Get ready to hack.</p>';
-
-echo '<p>Pssst.. any fmi ideas go <a href="https://fss.fmi.uni-sofia.bg/za-nas/kutiq-za-idei/" target="_blank">here</a></p>';
-
-echo "<table>";
-
-for($i = 1; $i < 10; $i++)
-{
-	echo "<tr>";
-	for($j = 1; $j < 10; $j++)
-    {
-	    if ($i == 1 || $j == 1)
-		{
-			echo "<th>".($i*$j)."</th>";
-		}
-		else
-		{
-			echo "<td>".($i*$j)."</td>";
-		}
-    }
-	echo "</tr>";
-}
-
-echo "</table>";
-
-?>
+<?php include './mult-table.php';?>
+
+<form>
+    <label for="course-title">Име на предмета</label>
+    <input type="text" id="course-title" name="title">
+	<br>
+	<label for="lecturer">Преподавател</label>
+	<input type="text" id="lecturer" name="lecturer">
+	<br>
+    <label for="description">Описание</label>
+    <input type="text" id="description" name="description">
+	<br>
+	Група
+	<br>
+	<label> <input type="radio" name="group" value="math">М</label>
+	<br>
+	<label> <input type="radio" name="group" value="appl-math">ПМ</label>
+	<br>
+	<label> <input type="radio" name="group" value="base-cs">ОКН</label>
+	<br>
+	<label> <input type="radio" name="group" value="core-cs">ЯКН</label>
+	<br>
+	<label for="credits">Кредити</label>
+	<input type="number" id="credits" name="credits">

```

Comment:
You could add here limit for minimum and maximum values. This way the user would need less attempts, clicks, thinking and time in order to fill the form correctly.